src/main/resources/resource/WebGui/app/service/js/IntroGui.js

```diff
@@ -1,5 +1,5 @@
-angular.module('mrlapp.service.IntroGui', []).controller('IntroGuiCtrl', ['$scope', '$log', 'mrl', '$timeout', function($scope, $log, mrl, $timeout) {
-    $log.info('IntroGuiCtrl')
+angular.module('mrlapp.service.IntroGui', []).controller('IntroGuiCtrl', ['$scope', 'mrl', '$timeout', function($scope, mrl, $timeout) {
+    console.info('IntroGuiCtrl')
```

Comment:
Is this log needed or only for debugging?

Reply:
meh... its on the edge of debug vs info
but the important is the removal of $log which provides nothing but a dependency on an old logging implementation